scripts/deploy.sh

```diff
@@ -3,4 +3,4 @@
 cd /home/dev/prodProxy
 git pull origin master
 npm run setup
```

Comment:
`npm run setup` could be part of the `server` instructions, so this can also be removed.